src/fmeval/eval_algorithms/qa_accuracy.py

```diff
@@ -215,6 +188,81 @@ def _quasi_exact_match_score(model_output: str, target_output: str) -> float:
 }
 
 
+def _get_score(
```

Comment:
Any particular reason to move this out of the class? I would say we keep it inside the class to encapsulate the logic around using `target_output_delimiter` inside the class only. 

We should be good to move it inside `QAAccuracyScores`. right?

Reply:
Sure, good point.